test/rates_test.rb

```diff
@@ -57,6 +57,9 @@ def test_exchange_rate
 
     assert_equal 5.944602, fx.exchange_rate(:from => "AUD", :to => "HRK")
     assert_equal 0.168220, fx.exchange_rate(:from => "HRK", :to => "AUD")
+
+    assert_equal 0.00023, fx.exchange_rate(:from => 'SLL', to: 'USD')
```

Comment:
New hash syntax would be preferred but project itself is not using it so old hash syntax was used. Is there any way to tell Hound about project preferences? :p